spec/hiptest_publisher_spec.rb

```diff
@@ -326,9 +326,16 @@ def expect_same_files(expected_directory, actual_directory)
 
       common_files = (actual_files & expected_files)
       common_files.each do |file|
-        actual_content = File.read("#{actual_directory}/#{file}")
-        expected_content = File.read("#{expected_directory}/#{file}")
-        expect(actual_content).to eq(expected_content), "File #{file} output is different from its expected output"
+        actual_file = "#{actual_directory}/#{file}"
+        expected_file = "#{actual_directory}/#{file}"
```

Comment:
Should `expected_file` and `actual_file` be referring to the same file? Should be:

```expected_file = "#{expected_directory}/#{file}"```